truffle-config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
Why commit code which is fully commented out?

Reply:
Excellent question, this is auto-generated code from the metacoin example truffle unbox, my preference would be for now to keep it, if needed for reference until a better understanding of the complete functionality is developed.